### PR TITLE
[Python] Populate setuptools description with README.md

### PR DESCRIPTION
**Pull request by leandron**
Adds the description metadata for the setuptools descriptor file `setup.py` with the contents of our existing README.md, which is a common practice.

This will be reflected in the page for our package on PyPI in pages like https://pypi.org/project/apache-tvm/0.9.dev1125/

cc @tqchen @areusch @Mousius @driazati @grant-arm for reviews

### CODE REVIEW

**Review by Mousius — APPROVED** [review]

**Comment by Mousius** [comment]
Much better than what we have today:

Screenshot 2022-04-20 at 16 43 50

**Review by grant-arm — APPROVED** [review]
LGTM!

File: python/setup.py [hunk 1]
```diff
@@ -217,6 +226,8 @@ def get_package_data_files():
     name="tvm",
     version=__version__,
     description="TVM: An End to End Tensor IR/DSL Stack for Deep Learning Systems",
+    long_description=long_description_contents(),
+    long_description_content_type="text/markdown",
```
Comment (driazati) [inline python/setup.py:230]: It's not strictly related but can we round out some other areas for PyPi while we're in here?
```suggestion
    long_description_content_type="text/markdown",
    url='https://tvm.apache.org/',
    download_url='https://github.com/apache/tvm/tags',
    author='Apache TVM',
    license='Apache',
    # See https://pypi.org/classifiers/
    classifiers=[
        "License :: OSI Approved :: Apache Software License",
        "Development Status :: 4 - Beta",
        "Intended Audience :: Developers",
        "Intended Audience :: Education",
        "Intended Audience :: Science/Research",
    ],
    keywords='machine learning',
```

File: python/setup.py [hunk 2]
```diff
@@ -201,6 +201,15 @@ def get_package_data_files():
     return ["relay/std/prelude.rly", "relay/std/core.rly"]
 
 
+def long_description_contents():
+    with open(
+        os.path.abspath(os.path.join(CURRENT_DIR, "..", "README.md")), encoding="utf-8"
```
Comment (driazati) [inline python/setup.py:206]: nit: `pathlib` makes life easier 

```suggestion
        pathlib.Path(CURRENT_DIR).resolve().parent / "README.md", encoding="utf-8"
```

**Review by driazati — APPROVED** [review]
just a couple tiny comments but 🚀

**Comment by areusch** [comment]
+1 for this, though agree with the others' comments and will let them approve

File: python/setup.py (hunk 1 above)
Comment (leandron) [reply python/setup.py:230]: Applied, thanks.

**Review by areusch — APPROVED** [review]

**Comment by leandron** [comment]
https://pypi.org/project/apache-tvm/0.9.dev1166/

**Pull request merged**